banzai/utils/image_utils.py

```diff
@@ -2,42 +2,28 @@
 from glob import glob
 import logging
 
-from astropy.io import fits
-
 from banzai import logs
 from banzai import dbs
+from banzai.utils.fits_utils import get_primary_header
 
 logger = logging.getLogger(__name__)
 
 
-def image_passes_criteria(filename, criteria, db_address=dbs._DEFAULT_DB):
-    instrument = dbs.get_instrument_for_file(filename, db_address=db_address)
-    return dbs.instrument_passes_criteria(instrument, criteria)
-
-
-def get_obstype(filename):
-    obstype = None
-    hdu_list = fits.open(filename)
-    for hdu in hdu_list:
-        if 'OBSTYPE' in hdu.header.keys():
-            obstype = hdu.header['OBSTYPE']
-    if obstype is None:
-        logger.error('Unable to get OBSTYPE', extra_tags={'filename': filename})
-    return obstype
+def image_can_be_processed(header, context):
+    passes = dbs.instrument_passes_criteria(header, context.FRAME_SELECTION_CRITERIA)
```

Comment:
`dbs.instrument_passes_criteria` needs to be updated to handle a header, rather than an instrument object

Reply:
Done.

Reply:
Also, if `instrument_passes_criteria` no longer queries the DB, then it should probably be moved out of `dbs.py`